subresource-loading.md

```diff
@@ -186,3 +192,132 @@ When returning to the event loop, if there are any chunk IDs on the list to fetc
 - Then, the original fetch can continue at the top of the algorithm. It should either be served by a mapped-in chunk or 404.
 
 *See the [subresource loading FAQ](./faq.md#subresource-loading) for more information.*
+
+#### Pros
+* Mixing in decelration of the resources in the bundle and scopes covered by it in ways that enable use of both techniques, without requiring hard-to-implement [Cache Digests](https://tools.ietf.org/html/draft-ietf-httpbis-cache-digest-05).
+* Decouples the resource number from the number of chunks in ways that enable higher scalability in cases of closely knit, very small modules.
+
+#### Cons
+* Requires declaring full dependency map on each entry point, that includes all the chunks required by all the entry points. If developer tooling will get that wrong, it can result in spurious downloads.
+* A "just download the entire bundle" fallback is not feasible, since the full bundle contains all of the code-split parts, but initial load should only load the necessary parts. Related, caches that ignores Vary will have have A Bad Time™. 
+* It could be prohibitively slow to dynamically compress collections of chunks to a high quality, as there are certain techniques that work quickly in the case of a large subset, but not as well in a smaller subset
+(See [results](https://dev.to/riknelix/fast-and-efficient-recompression-using-previous-compression-artifacts-47g5#results) for 90% removal)
+* It's improbable that all entry points require the same order of resources. This proposal doesn't provide a way for bundlers to address this and enable different chunk order for different entry points.
+
+
+## Design FAQ
```

Comment:
The FAQ was moved to faq.md; if you added new FAQs, best to add them over there.

Reply:
Moved